x-pack/plugins/secops/server/lib/framework/kibana_framework_adapter.ts

```diff
@@ -82,6 +83,26 @@ export class KibanaBackendFrameworkAdapter implements FrameworkAdapter {
       plugin: graphiqlHapi,
     });
   }
+
+  public getIndexPatternsService(
+    request: FrameworkRequest<Request>
+  ): FrameworkIndexPatternsService {
+    if (!isServerWithIndexPatternsServiceFactory(this.server)) {
+      throw new Error('Failed to access indexPatternsService for the request');
+    }
+    return this.server.indexPatternsServiceFactory({
+      // tslint:disable-next-line:no-any
+      callCluster: async (method: string, args: [object], ...rest: any[]) => {
```

Comment:
huh...Does the internal `callWithRequest` dynamically check types on the `...rest`?

If it's just strings or one type of data could we remove `any` with this?
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-0.html#rest-elements-in-tuple-types

Reply:
https://github.com/elastic/ingest-dev/issues/118